testsuite/MDAnalysisTests/converters/test_openmm_parser.py

```diff
@@ -130,6 +155,72 @@ class TestOpenMMTopologyParser(OpenMMTopologyBase):
     expected_n_bonds = 1922
 
 
+class TestOpenMMTopologyParserWithNoElements(OpenMMTopologyBase):
+    ref_filename = app.PDBFile(PDB).topology
+    expected_n_atoms = 47681
+    expected_n_residues = 11302
+    expected_n_segments = 1
+    expected_n_bonds = 25533
+
+    expected_attrs = [
+        "ids",
+        "names",
+        "resids",
+        "resnames",
+        "masses",
+        "bonds",
+        "chainIDs",
+    ]
+
+    @pytest.fixture()
+    def top(self, filename):
+        omm_topology = self.ref_filename
+        for a in omm_topology.atoms():
+            a.element = None
+        return self.parser(PDB) \
+                   ._mda_topology_from_omm_topology(omm_topology)
```

Comment:
This line continuation breaks is more of a PEP8 break than having a it on a single line.

Additionally do you really need to redefine `top` this way? Couldn't you just do:

```python
ref_filename = app.PDBFile(PDB).topology
for a in ref_filename.atoms():
    a.element = None
```
Replacing line 159?